R/Spectra.R

```diff
@@ -29,9 +32,14 @@ setMethod("saveMsObject", signature(object = "Spectra",
           })
 
 #' @rdname PlainTextParam
+#'
+#' @importFrom methods getFunction
 setMethod("readMsObject", signature(object = "Spectra",
                                    param = "PlainTextParam"),
-          function(object, param, spectraPath = character()) {
+          function(object, param, ...) {
+              if (!requireNamespace("Spectra", quietly = TRUE))
```

Comment:
Why do we do this here but not in the MsBackendMzR code ?

Reply:
because user will prob never only save the backend ? and always go through Spectra ?

Reply:
hmmmm. good question. I had some weird errors and then added this - I guess it's not needed in the end. I can remove.

Reply:
removed it in the last commit